src/common/uiactioncmn.cpp

```diff
@@ -203,13 +203,21 @@ bool wxUIActionSimulator::Select(const wxString& text)
 #endif // wxUSE_COMBOBOX
 #if wxUSE_CHOICE
     wxChoice* choice;
-    if ( !container && (choice = wxDynamicCast(focus, wxChoice)) )
-        container = choice;
+    if ( !container )
+    {
+        choice = wxDynamicCast(focus, wxChoice);
+        if ( choice )
+            container = choice;
```

Comment:
I'd write this as

```suggestion
        if ( auto choice = wxDynamicCast(focus, wxChoice) )
            container = choice;
```

(and remove `wxChoice* choice` declaration above, of course).

Reply:
git config user.email was actually empty so thanks for informing me, I have it fixed now. as for the code I will make some changes then and sort out this strange commit issue.